04_data-viz.qmd

```diff
@@ -391,40 +436,56 @@ tibble(x = 1:100,
 
 ```
 
-### `r kableExtra::text_spec("Exercise 5", color = "#1696d2")`
 
-**`r kableExtra::text_spec("Step 1:", color = "#55b748")`** Create a scatter plot of the `storms` data set with `pressure` on the x-axis and `wind` on the y-axis. 
+::: callout
 
-**`r kableExtra::text_spec("Step 2:", color = "#55b748")`** Add `facet_wrap(~ category)`
+#### [`r paste("Exercise", exercise_number)`]{style="color:#1696d2;"}
 
-\vspace{0.25in}
+```{r}
+#| echo: false
+
+exercise_number <- exercise_number + 1
+```
+
+1. Create a scatter plot of the `storms` data set with `pressure` on the x-axis and `wind` on the y-axis. 
+
+2. Add `facet_wrap(~ category)`
+
+:::
+
+### Facets:
 
 :::{.callout-note}
-**6 Facets (optional)** break data into meaningful subsets. `facet_wrap()`, `facet_grid()`, and `facet_geo()`.
+**Facets (optional)** break data into meaningful subsets. `facet_wrap()`, `facet_grid()`, and `facet_geo()`.
 :::
 
-### Facet wrap
+`facet_wrap` and `facet_grid` are similar but have distinct functions. As the diagram below illustrates, `facet_wrap` organizes a 1 dimensional set of charts, but wraps the charts in two dimensions to preserve space. `facet_grid` organizes a set of charts in two dimensions. `facet_geo` organizes charts in a way that attempts to preserve some geographic component of the data. 
+
+![Source: [ggplot2: Elegant Graphics for Data Analysis (3e)](https://ggplot2-book.org/facet)](images/position-facets.png)
```

Comment:
Does this fit the license of ggplot2: Elegant Graphics for Data Analysis (3e)? I don't want to include content from elsewhere unless it is legal.

Reply:
So somewhat shockingly the [ggplot2 book](https://ggplot2-book.org) doesn't have a LICENSE page. I took it out and added a link to be safe.